serverless/architecture/serverless-serving-architecture.adoc

```diff
@@ -0,0 +1,26 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-serving-architecture"]
+= Knative Serving architecture
+:context: serverless-serving-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Serving supports deploying and managing serverless applications, by providing a set of objects as Kubernetes custom resource definitions (CRDs), or _resources_, that are used to define and control the behavior of serverless workloads on an {product-title} cluster.
+
+These CRDs are building blocks to address complex use cases, for example:
+
+* Rapidly deploying serverless containers.
+* Automatically scaling pods.
+// TODO: Add other use cases, more advanced tutorials?
+
+== Knative Serving resources
+// TODO: Request a good Serving arch diagram
+
+Configuration:: The `configuration.serving.knative.dev` resource, or Configuration, maintains the desired state for your deployment. It provides a clean separation between code and configuration. Modifying a configuration creates a new Revision.
+
+Revision:: The `revision.serving.knative.dev` resource, or Revision,  is a point-in-time snapshot of the code and configuration for each modification made to the workload. Revisions are immutable objects and can be retained for as long as necessary.
+
+Service:: The `service.serving.knative.dev` resource, or Service, automatically manages the life cycle of your workload. It controls the creation of other objects to ensure that your application has a Route, a Configuration, and a new Revision for each change to the Service. Services enable traffic routing to your chosen Revision.
```

Comment:
Does repeating _Service_ in the text have some significance? It's also the definition name for this text.

Reply:
Good point.